docs/sources/datasources/azuremonitor/template-variables.md

```diff
@@ -34,6 +34,7 @@ The Azure Monitor data source provides the following queries you can specify in
 | Metric Names    | Returns a list of metric names for a resource.                                               |
 | Workspaces      | Returns a list of workspaces for the specified subscription.                                 |
 | Logs            | Use a KQL query to return values.                                                            |
+| Resource Graph  | Use an ARG query to return values.                                                           |
```

Comment:
Any preference on the naming? 

(Azure) Resource Graph

Reply:
I think I'd err on the side of using the full name for clarity 😊

Reply:
I'd use `Resource Graph`, saying `Azure` again is redundant but then in the query editor, we say "Azure Resource Graph"...

![Screenshot from 2022-08-02 16-13-57](https://user-images.githubusercontent.com/4025665/182396236-ecfc33a7-01fc-410f-95a5-c4fc41ce0e84.png)

I'd still go with `Resource Graph` but both options are fine.